docs/design/concurrent-merge-split.md

```
@@ -0,0 +1,206 @@
+            break
+        }
+        next = t.findFloorNode(next.InsNextID)
+    }
+}
+```
+
+This only applies to element nodes. Text splits use offset-based IDs with the
+same `CreatedAt`, so `findFloorNode` already resolves them correctly.
+
+**Affected tests**: #4 contained-split-and-split-at-different-levels,
+#5 contained-split-and-delete-the-whole, #9 side-by-side-split-and-insert,
```

markdownlint-cli2 (0.22.0)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)